src/thunderbots/software/test/world/ball.cpp

```diff
@@ -2,42 +2,47 @@
 
 #include <gtest/gtest.h>
 
-using namespace std::chrono;
-
 class BallTest : public ::testing::Test
 {
    protected:
-    void SetUp() override
+    // We need to provide a constructor that initializes the Timestamp variables
+    // since the Timestamp class has no default constructor
```

Comment:
Thoughts on giving it a default constructor?

Reply:
Gave it a default constructor